addons/survey/views/survey_templates_statistics.xml

```diff
@@ -217,13 +220,20 @@
                                 </t>
                             </td>
                             <td>
-                                <t t-if="question.question_type == 'text_box'">
-                                    <span t-if="no_print_url" t-field="input_line.value_text_box" />
-                                    <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_text_box"></a>
+                                <t t-set="value_field" t-value="input_line._get_answer_value_field()"/>
+                                <!-- If answer already covered by filter or if there is only one line to display, do not allow filtering on it again -->
+                                <t t-if="question.id in already_filtered_questions or len(table_data) == 1">
+                                    <t t-esc="input_line[value_field]"/>
                                 </t>
-                                <t t-if="question.question_type == 'char_box'">
-                                    <span t-if="no_print_url" t-field="input_line.value_char_box" />
-                                    <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_char_box"></a>
+                                <t t-else="">
+                                    <a class="text-primary filter-add-answer"
+                                        data-model-short-key="L" t-att-data-record-id="input_line.id"
+                                        role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer">
+                                        <t t-esc="input_line[value_field]"/>
+                                    </a>
+                                    <i class="fa fa-filter text-primary ps-1 filter-add-answer"
+                                        data-model-short-key="L" t-att-data-record-id="input_line.id"
+                                        role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer"/>
```

Comment:
Seems like we are repeating a bit the same thing.

Can't we wrap the "i" tag into the "a" tag to avoid having to set the same attributes (data-*, title, ...- multiple times?

(Same fort others).